bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -92,10 +92,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             if (channelType.contains(CHANNEL_SWITCH)) {
                 logger.debug("A command was sent to a sensor type so we are ignoring the command");
             } else {
-                int sendCommand = (OnOffType.OFF.compareTo((OnOffType) command));
-                logger.debug("The command being sent to zone {} for channel:{}  is {}", zone, channelUID.getAsString(),
```

Comment:
any reason you are taking these logging statements out?  I get not wanting to clutter up the logs.  But i think maybe we keep them in but up the logger to logger.trace? 

Just so that they are still available for future debugging?

Reply:
ok that makes sense.